docs/tutorials/autox/autox.md

```diff
@@ -52,7 +52,7 @@ kubectl label deployment httpbin app.kubernetes.io/version=1.0.0
 Next, we will configure and install the AutoX controller.
 
 ```bash
-helm install autox autox --repo https://iter8-tools.github.io/hub/ --version 0.1.6 \
+helm install autox autox --repo https://iter8-tools.github.io/iter8 --version 0.1.6 \
```

Comment:
@Alan-Cha @sriumcp We've restricted the version in all the autox examples but not others. Why?

Reply:
It is generally a good idea to do version pinning like this (for e.g., if we release a new autox chart with breaking changes, this documentation will still be valid).

I am not sure why we are inconsistent.